src/pymodaq_plugins_keithley/utils.py

```diff
@@ -1,15 +1,34 @@
 # -*- coding: utf-8 -*-
 """
-Created the 31/08/2023
+Created the 31/08/2023, updated the 19/06/2024
 
-@author: Sebastien Weber
+@author: Sebastien Weber, Sebastien Guerrero
 """
+import os
 from pathlib import Path
-
+from pymodaq.utils.logger import set_logger
 from pymodaq.utils.config import BaseConfig, USER
-
+import pymodaq_plugins_keithley as plugin
+logger = set_logger('plugin.utils', add_to_console=False)
 
 class Config(BaseConfig):
     """Main class to deal with configuration values for this plugin"""
     config_template_path = Path(__file__).parent.joinpath('resources/config_template.toml')
     config_name = f"config_{__package__.split('pymodaq_plugins_')[1]}"
+
+class Config_keithley(BaseConfig):
```

Comment:
use the basic cplugin config file and add into it sections

Reply:
Done (new config removed)